### PR TITLE
Add `grid-calc` to abstract the calculation of column widths

### PR DESCRIPTION
**Pull request by whmii**
No visual changes.

### CODE REVIEW

**Comment by whmii** [comment]
Great job Will!

**Comment by whmii** [comment]
Thanks!

**Comment by whmii** [comment]
You're soo cool

File: core/neat/functions/_grid-calc.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+/// @argument {number} $columns
+///
+/// @return {string}
+///
+/// @example scss
+///   _grid-calc($neat-grid, 4)
+///
+/// @access private
+
+@function grid-calc($grid, $columns) {
```
Comment (tysongach) [inline core/neat/functions/_grid-calc.scss]: Should `$grid` have a default of `$neat-grid`?

Comment (tysongach) [inline core/neat/functions/_grid-calc.scss]: If this function is private, as indicated on Line 13, it should probably be named `_grid-calc` (with an underscore).

**Pull request merged**